static/js/unscrambleEmail.js

```diff
@@ -0,0 +1,21 @@
+import { unscramble } from "botex";
+
+export function onRouteDidUpdate({ location, previousLocation }) {
+  if (typeof window !== "undefined") {
```

Comment:
```suggestion
  if (ExecutionEnvironment.canUseDOM) {
```

You'd also need to add `import ExecutionEnvironment from '@docusaurus/ExecutionEnvironment';`.